libs/core/src/lib/upload-collection/upload-collection-button-group.component.ts

```diff
@@ -0,0 +1,86 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output, ViewEncapsulation } from '@angular/core';
+
+@Component({
+    selector: 'fd-upload-collection-button-group',
+    host: { class: 'fd-upload-collection__button-group' },
+    templateUrl: './upload-collection-button-group.component.html',
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class UploadCollectionButtonGroupComponent {
+    /** @hidden */
+    editMode = false;
+
+    /** @hidden */
+    okDisabled = false;
```

Comment:
Good point but it is intended to be used by the dev so i will remove the hidden tag

Reply:
The value of this property should actually be handled from the parent component, I forgot to finish implementing this

Reply:
I see the component is exposed and intended to be used by the customer, it has public properties. So when the user will work with the component it will think that not underscored properties are public. As we're the library developers and we definitely know how to work with component we're okay with using underscored properties between components.